pontoon/pretranslation/transformer.py

```diff
@@ -93,6 +95,48 @@ def create_locale_plural_variants(node: FTL.SelectExpression, locale: Locale):
     node.variants = variants
 
 
+def extract_accesskey_candidates(message: FTL.Message, label: str):
+    def get_source(names):
+        for attribute in message.attributes:
+            if attribute.id.name in names:
+                element = attribute.value.elements[0]
+
+                if isinstance(element, FTL.TextElement):
+                    return element.value
+                elif isinstance(element.expression, FTL.SelectExpression):
+                    # We only need the first candidate,
+                    # so it's enough to just use the first variant
+                    variant = element.expression.variants[0]
+                    return variant.value.elements[0].value
```

Comment:
Two issues with this:
1. You should check for `isinstance(variant.value.elements[0], FTL.TextElement)`.
2. This is sub-optimal when both the `.label` and `.accesskey` depend on `PLATFORM()`. For example, a label `Options`/`Preferences` might have an accesskey `O`/`P`, and this would pick the `windows` label to generate a `macos` accesskey.